projects/github-actions/repo-gardening/src/tasks/add-labels/index.js

```diff
@@ -225,6 +225,11 @@ async function getLabelsToAdd( octokit, owner, repo, number, isDraft ) {
 			keywords.add( 'E2E Tests' );
 		}
 
+		const anyTestFile = file.match( /\/tests\// );
```

Comment:
You might want to also check for `/test/`, which seems to be a common pattern for module and block tests.

Reply:
I opened #30051 to follow up on this.